pkg/secrets/manager.go

```diff
@@ -33,6 +33,7 @@ var (
 		},
 	)
 	secretsTotal = prometheus.NewGauge(
+		//nolint:promlinter
 		prometheus.GaugeOpts{
 			Name: "prometheus_kubernetes_secrets_total",
```

Comment:
@TheSpiritXIII This got flagged by promlinter.
`non-counter metrics should not have "_total" suffix`

Suggestions?

Reply:
I'm not sure we can change the metric type 🤔 . Can we rename it to `prometheus_google_kubernetes_secrets_total` since it's for our fork, and turn it into a Counter?

Reply:
Good question, already benefit of having this promlinter enabled!

Shouldn't it be a gauge of `prometheus_kubernetes_secrets_configured`?  and `prometheus_kubernetes_failed_secret_configs` I would do `prometheus_kubernetes_secrets_misconfigured`?

It's safe to change here quickly